draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -719,14 +723,12 @@ of the original packet stream cannot be re-constructed from the C-DNS format:
      * A DNS payload may have be split across multiple TCP segments
      * Multiple DNS messages may have be sent on a single TCP session
 * Malformed DNS messages if they are not recorded
-* Non-DNS packets
```

Comment:
But that assumes the collecting program has an option to 'ignore traffic arriving on the ports specified for capture if it doesn't meet these conditions'. If it doesn't, then Non-DNS traffic doesn't exist. It's either DNS traffic or it's malformed DNS traffic, and those we've already covered. That's why I think this is implementation-dependent.

I guess we could add an item saying 'Traffic discarded by an implementation for implementation-specific reasons'?